arangod/Aql/ExecutorInfos.cpp

```diff
@@ -25,12 +25,12 @@
 using namespace arangodb::aql;
 
 ExecutorInfos::ExecutorInfos(
-    std::shared_ptr<std::unordered_set<RegisterId>> inputRegisters,
-    std::shared_ptr<std::unordered_set<RegisterId>> outputRegisters,
+    std::shared_ptr<std::unordered_set<RegisterId>> readableInputRegisters,
```

Comment:
isn't readableInput doubled ?
input := read
output := write

Reply:
This the the solution @goedderz and I came up with after some discussion. It seemed to be the most correct way. As there are are for example input registers that are not readable. They just get copied as they might be in registers to keep.